cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -277,11 +277,6 @@ type AutoscalingOptions struct {
 	// dynamicNodeDeleteDelayAfterTaintEnabled is used to enable/disable dynamic adjustment of NodeDeleteDelayAfterTaint
 	// based on the latency between the CA and the api-server
 	DynamicNodeDeleteDelayAfterTaintEnabled bool
-	//IgnoreSchedulerProcessing is used to signal whether CA will/won't wait
-	//for scheduler to mark pods as unschedulable and will process both marked & non-marked pods
-	//it will also signal whether we enable/disable waiting for pod time buffers before triggering a scale-up.
-	IgnoreSchedulerProcessing bool
-	//IgnoredSchedulers are used to specify which schedulers to ignore their processing
-	//if IgnoreSchedulerProcessing is set to true
-	IgnoredSchedulers map[string]bool
+	//BypassedSchedulers are used to specify which schedulers to bypass their processing
```

Comment:
nit: Add a space after `//` to keep comments consistent.

Reply:
Done